### ISSUE TITLE
Ranged attack visibility

### ISSUE THREAD

**Issue by FusuMusu** [Grokmoo/sulis#205]
Hi, when fighting, if an enemy is in an area that is discovered but not visible, and in the range of a ranged attack (or might only be an archer) shouldn't it be that you can't see or attack the foe?

In my case the enemy disapears but if i hover over the tile it shows the name and life and all, and i can attack it with my archer, even when it's not in sight.

This happened in the prologue with Aessa with her crossbow, if it matters.

**Comment by Grokmoo** [comment]
Yeah, I have definitely seen this happen before when the line-of-sight calculations don't quite line up.
